pyaedt/hfss.py

```diff
@@ -3686,6 +3703,106 @@ def assign_impedance_to_sheet(self, sheet_name, sourcename=None, resistance=50,
             return self._create_boundary(sourcename, props, "Impedance")
         return False
 
+    @pyaedt_function_handler()
+    def assign_anisotropic_impedance_to_sheet(
+        self,
+        sheet_name,
+        sourcename=None,
+        ZxxResi=0,
+        ZxxReac=0,
+        ZxyResi=0,
+        ZxyReac=0,
+        ZyxResi=0,
+        ZyxReac=0,
+        ZyyResi=0,
+        ZyyReac=0,
```

Comment:
I would suggest to follow pythonic naming convention if possible.
For instance: `z_xx_resistance`